docs/getting-started/security-ui.asciidoc

```diff
@@ -32,6 +32,29 @@ The navigation menu contains direct links and expandable groups, identified by t
 [role="screenshot"]
 image::images/nav-overview.gif[Overview of the navigation menu]
 
+[float]
+[[inline-actions]]
+== Inline actions for fields and values
+
+Throughout the {security-app}, you can hover over many data fields and values to display inline actions, which allow you to customize your view or investigate further based on that field or value. 
+
+[role="screenshot"]
+image::images/inline-actions-menu.png[Inline additional actions menu,45%]
+
+In some visualizations, these actions are available in the legend by clicking a value's options icon (image:images/three-dot-icon-vertical.png[Vertical three-dot icon,16,16]).
+
+[role="screenshot"]
+image::images/inline-actions-legend.png[Actions in a visualization legend,90%]
+
+Inline actions include the following (some actions are unavailable in some contexts):
+
+* *Filter In*: Add a filter that includes the selected value.
+* *Filter Out*: Add a filter that excludes the selected value.
+* *Add to timeline*: Launch Timeline and populate a query with the selected value.
+* *Toggle column in table*: Add or remove the selected field as a column in the displayed alerts or events table. (This action is only available on an alert or event's details flyout.)
```

Comment:
```suggestion
* *Toggle column in table*: Add or remove the selected field as a column in a table. (This action is only available on an alert or event's details flyout.)
```
"displayed table" seemed off to me, and I think which table this option affects will be clear from context in practice

Reply:
Let me tinker and try to make this clearer. I'm not sure "a table" works because it's not just any table, it's the table that the user was working with (and which was displayed on screen) when they opened the details flyout.

Reply:
I think we can just remove "displayed." True, someone actually using the UI will probably understand the context, but someone reading this might need more specifics on which table is affected, since the whole overview page is somewhat context-less and generic:
```suggestion
* *Toggle column in table*: Add or remove the selected field as a column in the alerts or events table. (This action is only available on an alert or event's details flyout.)
```